Library/Homebrew/dev-cmd/tap-new.rb

```diff
@@ -105,8 +116,20 @@ def run
 
                 - run: brew test-bot --only-tap-syntax
 
+                - name: Base64-encode GITHUB_TOKEN
+                  id: base64-encode
+                  if: github.event_name == 'pull_request'
+                  env:
+                    TOKEN: ${{ github.token }}
+                  run: |
+                    base64_token=$(echo -n "${TOKEN}" | base64 | tr -d "\\n")
+                    echo "::add-mask::${base64_token}"
+                    echo "token=${base64_token}" >> "${GITHUB_OUTPUT}"
+
                 - run: brew test-bot --only-formulae#{" --root-url='#{root_url}'" if root_url}
                   if: github.event_name == 'pull_request'
+                  env:
+                    HOMEBREW_DOCKER_REGISTRY_TOKEN: ${{ steps.base64-encode.outputs.token }}
```

Comment:
Is this just a no-op if it's an empty value?

Reply:
Correct.
But I've wrapped it into `if args.github_packages?` so it won't be even set

Reply:
Wonderful.